zest/releaser/towncrier_hooks.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from zest.releaser import utils
+import logging
+import os
+import pkg_resources
+import toml
+
+# towncrier might become a conditional dependency of zest.releaser,
+# for example in the recommended list.
+# If we split this into a separate package, it should be a hard dependency.
+try:
+    pkg_resources.get_distribution('towncrier')
+except pkg_resources.DistributionNotFound:
+    HAS_TOWNCRIER = False
+else:
+    HAS_TOWNCRIER = True
+
+
+logger = logging.getLogger('towncrier_hooks')
+TOWNCRIER_MARKER = '_towncrier_applicable'
+TOWNCRIER_CONFIG_FILE = 'pyproject.toml'
+
+
+def load_config():
+    fn = os.path.join(TOWNCRIER_CONFIG_FILE)
+    if not os.path.exists(fn):
+        return
+    with open(fn, 'r') as conffile:
+        config = toml.load(conffile)
+    if 'tool' not in config:
+        return
+    config = config['tool']
+    if 'towncrier' not in config:
+        return
+    config = config['towncrier']
```

Comment:
Another one. Is this common practice for .toml files?

Reply:
Maybe. It is in towncrier.
It can be okay: you start with the complete config and drill down until you have the part of the config that you really need, and don't care about the abandoned parts.